pkgs/applications/misc/survex/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, autoreconfHook
+, pkg-config
+, wxGTK30-gtk3
+, ffmpeg
+, proj
+, perl532
+, unscii
+, python
+, libGL
+, libGLU
+, x11
+, docbook2x
+, docbook5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "survex";
+  version = "1.2.44";
+
+  nativeBuildInputs = [ docbook5 x11 libGL libGLU docbook2x autoreconfHook pkg-config wxGTK30-gtk3 ffmpeg proj python (perl532.withPackages (perlPackages: with perlPackages; [ LocalePO ] )) ];
+
+  src = builtins.fetchGit {
+    url = "git://git.survex.com/survex";
+    rev = "38bc41554efda8c15494ca44e5fc17c8146a077f";
+    ref = "refs/tags/${version}";
+  };
```

> while evaluating the attribute 'src' of the derivation 'survex-1.2.44' at <…>/pkgs/stdenv/generic/make-derivation.nix:201:11:
> access to URI 'git://git.survex.com/survex' is forbidden in restricted mode

```suggestion
  src = fetchgit {
    url = "git://git.survex.com/survex";
    rev = version;
    sha256 = "11gaqmabrf3av665jy3mr0m8hg76fmvnd0g3rghzmyh8d8v6xk34";
  };
```

There's a GitHub repo for this project ([ojwb/survex](https://github.com/ojwb/survex)), do you know if this is official? They have already tagged a version 1.2.45 there.

---

It is an official mirror, yes.

---

Then can we bump the package to 1.2.45 or does something prevent us from doing so?